json/json_stringify_stream.ts

```diff
@@ -82,6 +82,9 @@ export interface StringifyStreamOptions {
  * ```
  */
 export class JsonStringifyStream extends TransformStream<unknown, string> {
+  /**
+   * Construct new instance
+   */
```

Comment:
This should have documentaiton about the options that can be provided.

Reply:
done

Reply:
@crowlKats, `StringifyStreamOptions` is fully documented above. To clarify, did you mean to provide example snippets where the constructor is used in different ways? Is that required if we already have examples on the class above?